x/staging/subspaces/keeper/msg_server.go

```diff
@@ -28,7 +28,7 @@ func (k msgServer) CreateSubspace(goCtx context.Context, msg *types.MsgCreateSub
 	// Check the if the subspace already exists
 	if k.DoesSubspaceExist(ctx, msg.SubspaceID) {
 		return nil,
-			sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "the subspaces with id %s already exists", msg.SubspaceID)
+			sdkerrors.Wrapf(types.ErrSubspaceAlreadyExistent, msg.SubspaceID)
```

Comment:
Can we rename this to be `ErrDuplicatedSubspace` to make it coherent with other errors?